### ISSUE TITLE
link rel="fluid-icon" should be treated as rel="icon"

### ISSUE THREAD

**Issue by maxnikulin** [danny0838/webscrapbook#168]
To check the current shape of webscrapbook I have captured a fragment from https://github.com/danny0838/webscrapbook/wiki/Backend I have surprised that uMatrix shows blocked cookies when I open the captured page.

The problem is that the following URL is not captured
```html
<link rel="fluid-icon" href="https://github.com/fluidicon.png" title="GitHub">
```
I was unaware of such option for the `rel` attribute. It seems that firefox prefers it as favicon or something else (e.g. in new tab interface for frequently visited sites). Other variants mentioned in https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/UI_considerations#Rich_icons are considered as icons candidates as well:
- `apple-touch-icon`
- `apple-touch-icon-precomposed`

Quick search gives impression that it is something for Mac to e.g. create shortcuts for desktop but firefox fetches them even on Linux. Issue associated with patch https://hg.mozilla.org/mozilla-central/rev/b11aa832c41ac5beef9065f804d11fb7c9887990 has no details related to particular attribute values.

Captured page contains the following links as well
```
<link rel="dns-prefetch" href="https://avatars3.githubusercontent.com/" data-scrapbook-orig-attr-href-20200825061754000="https://avatars3.githubusercontent.com">
```
I think, it is better to drop them completely or at least comment them out.

P.S. In general the extension looks promising

**Comment by danny0838** [comment]
Currently only a `rel` containing `icon` is supported. There are so many self-invented icon protocols created in recent years, and saving all alternative icons may be a waste. If there's a complete list we may consider adding an option like "save all favicons" for it.

**Comment by maxnikulin** [comment]
I totally agree that such non-standard stuff is disappointing. I have not checked chromium, MDN says that firefox could choose among that 3 options and usual "icon" to find the "best" one. Firefox has fallback to screenshot it it could not find appropriate image for top sites (e.g. files were removed from the server since page has been captured). So missed file should not be a real issue. However it is better to avoid unnecessary requests to external resources from captured pages.

I believe it is better to have an option to capture all icons in additional to capture the best one (some explicit scoring will be required).

What do you think concerning an option to manage the policy applicable to non-categorized links (keep or blank hrefs for unknown icons, DNS prefetch URLs, etc.)?

P.S. Concerning rich metadata, I could dream of gallery view in some future version of ScrapBook with images taken from `og:image` or `twitter:image` `meta` elements.
